src/Microsoft.Extensions.OData.Migration/EdmExtensions.cs

```diff
@@ -0,0 +1,463 @@
+﻿// ------------------------------------------------------------------------------
+// <copyright company="Microsoft Corporation">
+//     Copyright © Microsoft Corporation. All rights reserved.
+// </copyright>
+// ------------------------------------------------------------------------------
+
+namespace Microsoft.Extensions.OData.Migration
+{
+    using Microsoft.AspNet.OData;
+    using Microsoft.AspNet.OData.Routing;
+    using Microsoft.OData.Edm;
+    using Microsoft.Spatial;
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.IO;
+    using System.Linq;
+    using System.Reflection;
+    using System.Xml.Linq;
```

Comment:
Please sort the using. 

1. Move the System.* at head
2. Alphabet the others

Reply:
changed